datafusion/src/optimizer/simplify_expressions.rs

```diff
@@ -0,0 +1,538 @@
+// regarding copyright ownership.  The ASF licenses this file
```

Comment:
```suggestion
// Licensed to the Apache Software Foundation (ASF) under one
// or more contributor license agreements.  See the NOTICE file
// distributed with this work for additional information
// regarding copyright ownership.  The ASF licenses this file
```

The first few lines seem to be missing from the RAT